config/routes.rb

```diff
@@ -154,11 +154,15 @@
     resources :reads, only: [:create]
   end
 
+  namespace :webhooks do
+    post "/mailchimp/:secret/unsubscribe", to: "mailchimp_unsubscribes#create", as: :mailchimp_unsubscribe
+  end
+
   resources :messages, only: [:create]
   resources :chat_channels, only: %i[index show create update]
   resources :chat_channel_memberships, only: %i[create update destroy]
   resources :articles, only: %i[update create destroy]
-  resources :article_mutes, only: %i[update]
+  resources :article_mutes,  only: %i[update]
```

Comment:
weird that it added a second space here

Reply:
Maybe I fat-fingered something, I'll remove the extra space.